src/ext.js

```diff
@@ -203,10 +203,18 @@ export default function ext({ translator, shouldHide, senseNavigation, theme, is
                 },
                 chartId: {
                   type: "string",
-                  expression: "optional",
                   ref: "navigation.chartId",
                   translation: "properties.chartPicker.chartId",
+                  component: "expression-with-dropdown",
+                  expressionType: "StringExpression",
                   show: (data) => (isGoToChartEnabled ? checkShowNavigation(data, "chartId") : false),
+                  options: async (data, action, hyperCubeHandler) => {
+                    const sheetId = data.navigation.sheet;
+                    const objects = sheetId && (await hyperCubeHandler.app.getObject(sheetId));
+                    return objects?.properties.cells.map((cell) => ({
```

Comment:
For clarity, this should be called just `object` or even `sheet`, to no be confused with the objects _on_ the sheet.

Reply:
Umm... here I'm getting the objects on the sheet. If the sheet has three charts that means it contains three objects, doesn't it ?

Reply:
Ok, I'm thinking, you're right it's actually a sheet. I'll name it 'sheet'.